src/Symfony/Component/HttpFoundation/ResponseHeaderBag.php

```diff
@@ -281,7 +281,7 @@ public function makeDisposition($disposition, $filename, $filenameFallback = '')
     protected function computeCacheControlValue()
     {
         if (!$this->cacheControl && !$this->has('ETag') && !$this->has('Last-Modified') && !$this->has('Expires')) {
-            return 'no-cache';
+            return 'no-cache, private';
```

Comment:
Adding `private` is what we are doing by default line 299 in the same situation.